notebooks/yolov11-optimization/yolov11-object-detection.ipynb

```diff
@@ -1007,7 +1007,7 @@
     "    skip_first_frames=0,\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        # video_width=1280,
</pre><p>is it commented only for testing or you plan to stay in this way?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
I left it just as a hint